docker/irods_clients/ubuntu/Dockerfile

```diff
@@ -0,0 +1,153 @@
+ARG BASE_IMAGE=ubuntu:bionic
+FROM $BASE_IMAGE as installer
+
+# Other iRODS versions available on bionic are 4.3.0
+ARG IRODS_VERSION="4.2.11"
+ARG BATON_VERSION="4.0.0"
+ARG HTSLIB_VERSION="1.16"
+ARG SAMTOOLS_VERSION="1.16.1"
+ARG HTSLIB_PLUGINS_VERSION="201712"
+
+COPY . /opt/docker/irods_clients
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    apt-get update && \
+    apt-get install -q -y --no-install-recommends \
+    apt-utils \
+    ca-certificates \
+    curl \
+    dirmngr \
+    gpg \
+    gpg-agent \
+    lsb-release \
+    locales && \
+    locale-gen en_GB en_GB.UTF-8 && \
+    localedef -i en_GB -c -f UTF-8 -A /usr/share/locale/locale.alias en_GB.UTF-8
+
+ENV LANG=en_GB.UTF-8 \
+    LANGUAGE=en_GB \
+    LC_ALL=en_GB.UTF-8
+
+RUN curl -sSL https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
```

Comment:
I have seen people using `curl -fsSL` when getting keys. To fail without output. But then you may need a pipefail I think.

Reply:
We'd also need to change the default Dockerfile `RUN` shell from `/bin/sh` to `bash`, to allow pipefail. I don't feel that it's work the extra complexity. We'd exchange apt-key failing on an HTML error response to it failing on a empty response, without the exact cause being any more visible in the build log.